client/js/verification-portal/pregnancy-resource-center/get-resource-to-verify-button.js

```diff
@@ -180,11 +181,16 @@ const GetResourceToVerifyButton = ({ dispatch, changeFieldValue }) => {
 							changeFieldValue,
 							resource: store.getState().resource,
 						})
+						console.log('state', store.getState())
+						history.push(
+							'/verification/pregnancy-resource-center/' +
+								store.getState().resource._id,
```

Comment:
What do you think about changing this to a template string? i.e.
`/verification/...${store.getState...}`